### PR TITLE
Theming v2 channel search react

### PR DESCRIPTION
**Pull request by MartinCupela**
### 🎯 Goal

Add channel search styles for theme v2 as per https://github.com/GetStream/stream-chat-react/pull/1685.

### CODE REVIEW

File: src/v2/styles/ChannelSearch/ChannelSearch-theme.scss [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+  --str-chat__channel-search-input-wrapper-border-block-end: var(--str-chat__surface-color) 1px solid;
+
+  /* Left (right in RTL layout) border of the component */
+  --str-chat__channel-search-input-wrapper-border-inline-start: var(--str-chat__surface-color) 1px solid;
+
+  /* Right (left in RTL layout) border of the component */
+  --str-chat__channel-search-input-wrapper-border-inline-end: var(--str-chat__surface-color) 1px solid;
+
+  /* The border radius used for the borders of the component */
+  --str-chat__channel-search-input-wrapper-active-border-radius: 20px;
```
Comment (szuperaz) [inline src/v2/styles/ChannelSearch/ChannelSearch-theme.scss:45]: I think it's better to use a theme variable for border-radius so integrators can easily customize it, we have `--str-chat__border-radius-md` which is 18px, if that's acceptable, or we can create a new one like `--str-chat__border-radius-lg` for 20px

Comment (MartinCupela) [reply src/v2/styles/ChannelSearch/ChannelSearch-theme.scss]: Thank you @szuperaz :+1: I have added a variable with the suffix `-lg`

**Review by szuperaz — APPROVED** [review]

**Pull request merged**